machines/Issuers/IssuersActions.ts

```diff
@@ -40,18 +40,6 @@ export const IssuersActions = (model: any) => {
           isVerified: false,
         }),
     }),
-    sendVerificationError: send(
-      (context: any, _event) => {
-        return {
-          type: 'VERIFY_VC_FAILED',
-          errorMessage: context.errorMessage,
-          vcMetadata: context.vcMetadata,
-        };
-      },
-      {
-        to: context => context.serviceRefs.vcMeta,
-      },
-    ),
     setErrorAsVerificationError: assign({
```

Comment:
@balachandarg-tw We are not using this action anywhere can you remove this also

Reply:
Addressed